docs/release/releasing.md

```diff
@@ -1,19 +1,17 @@
 # Releasing
+1. Ensure that the next version maintains SemVer:
+    * Check if `Unreleased` changes need a MINOR or MAJOR bump.
+    * If yes, then mark the next version as per [`gradle-release` plugin] docs.
```

Comment:
IMO we can automate it in Github Actions.

Reply:
Ideally, each pull request should be able to bump it, but tags cannot be parts of PRs unfortunately.
So GHA could have an input with `version: 1.1.0` or with `bump:minor`.

Reply:
IMO it's just git tag `version-alpha` && git push origin version-alpha

Reply:
Or we could say bye bye to axion plugin and hold the versions in the code rather than tags.

Reply:
> IMO it's just git tag version-alpha && git push origin version-alpha

Locally I also use `git` directly, circumventing the `./gradlew markNextVersion`. Documenting the command would be quite verbose, because the `$version` is a variable and the `$remote` is `origin` for non-forkers and `upstream` for forkers (by convention).

Reply:
Axion has this `./gradlew markNextVersion -Prelease.incrementer=incrementMinor` ✨ 
I'll go for the `bump` input in GHA

Reply:
Done, although the video is out of date now 😁
The GUI flow is the same, but the example values are different